packages/data/addon/mirage/routes/bard-lite.js

```diff
@@ -187,13 +186,16 @@ export default function(
         let dimensionValues = _getDimensionValues(dimension, dimensionFilter);
 
         return newRows.concat(
-          dimensionValues.map(value =>
-            // TODO figure out why Object.assign refuses to work in Phantom even with Babel polyfill
-            assign({}, currentRow, {
-              [`${dimension}|id`]: value.id,
-              [`${dimension}|desc`]: value.description
-            })
-          )
+          dimensionValues.map(value => {
+            Object.keys(value).forEach(key => {
+              if (key === 'description') {
+                currentRow[`${dimension}|desc`] = value[key];
+              } else {
+                currentRow[`${dimension}|${key}`] = value[key];
+              }
+            });
+            return currentRow;
```

Comment:
Won't this result in multiple copies of `currentRow` being returned in the `dimensionValues.map`?